impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -263,7 +263,8 @@ Entity ApplyBlurStyle(FilterContents::BlurStyle blur_style,
           Entity::FromSnapshot(input_snapshot, entity.GetBlendMode());
       Entity result;
       Matrix blurred_transform = blur_entity.GetTransform();
-      Matrix snapshot_transform = snapshot_entity.GetTransform();
+      Matrix snapshot_transform =
```

Comment:
Don't these modifications need to be applied to the other blur styles?

Reply:
Nope, the solid blur style is special since it draws the original snapshot again.  Since we are changing the coordinates the snapshot is made in we had to account for it.  We test the other blur styles and they aren't showing up any differences.

example:
**impeller_Play_AiksTest_MaskBlurVariantTestOuterOpaqueWithBlurImageFilter_Metal**
![impeller_Play_AiksTest_MaskBlurVariantTestOuterOpaqueWithBlurImageFilter_Metal](https://github.com/flutter/engine/assets/30870216/ba822536-ae26-4549-bc3d-de799c49d355)

Reply:
Ah, I see. You are adjusting the "draw over the top" copy to match here?

Reply:
Yep